Dockerfile.assisted-service-build

```
@@ -6,7 +6,7 @@ ENV PATH=$PATH:$GOROOT/bin:$GOPATH/bin
 
 USER 0
 
-RUN yum install -y epel-release && yum install -y gcc genisoimage jq git
+RUN yum install -y epel-release && yum install -y gcc genisoimage jq git && yum install libvirt-client libvirt-devel -y
```

why do you need another command? any specific reason not to do
```suggestion
RUN yum install -y epel-release && yum install -y gcc genisoimage jq git libvirt-client libvirt-devel
```